firmware/controllers/settings.cpp

```diff
@@ -1109,7 +1109,7 @@ void setDateTime(const char * const isoDateTime) {
 #if EFI_RTC
 	if (strlen(isoDateTime) > 0) {
 		efidatetime_t dateTime;
-		if (sscanf("%04u-%02u-%02uT%02u:%02u:%02u", isoDateTime,
+		if (sscanf(isoDateTime, "%4lu-%2hhu-%2hhuT%2hhu:%2hhu:%2hhu",
```

Comment:
This still doesn't parse correctly in the built firmware, though my test code parses the string output by `console` just fine.  Only the year and month will parse no matter what I try.